libraries/ms-common/inc/stm32f0xx/gpio_cfg.h

```diff
@@ -0,0 +1,4 @@
+#pragma once
```

Comment:
I think _cfg should be used if the file is used solely for configuration data and _mcu should be used if it contains MCU-specific implementations or logic, but it can also contain configuration data. Do you think that makes sense?